src/python/pants/backend/python/dependency_inference/default_unowned_dependencies.py

```diff
@@ -0,0 +1,316 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+_STDLIB_MODULES = [
```

Comment:
Are there any modules that were removed from previous versions and might not appear? :thinking:

Reply:
@Eric-Arellano will likely have thoughts on this. We actually... removed per-interpreter-version lists a while back in #12818. But that was because we were using it to _filter_ things, rather than to backstop a validation like this one.

Reply:
Yeah I think this is an ever-growing list unless there's a good reason why it shouldn't be.

My question here is basically I just dumped the std library list form 3.10 because it was easy, so is there anything that needs to be added because 3.10 doesn't have them (anymore).